src/main/java/net/mcreator/util/image/ImageUtils.java

```diff
@@ -66,17 +67,17 @@ public static float getAverageLuminance(BufferedImage image) {
 		return luminance / (float) pixelCount;
 	}
 
-	public static ImageIcon drawOver(ImageIcon i, ImageIcon wh) {
+	public static ImageIcon drawOver(ImageIcon i, @Nullable ImageIcon wh) {

```

Comment:
What is the purpose of drawOver if the second image is null. If you want to resize image, use resize method, not drawOver

Reply:
I'm not resizing the image, I'm doing exactly how it was done to show the preview of the armor texture in the Armor mod element

Reply:
What does call to drawOver with second argument null do?

Reply:
Still wondering 

> What does call to drawOver with second argument null do?

as now you just inlined the code, but I am still not sure what that code is trying to achieve

Reply:
Isn't it obvious? The code display the preview of the villager texture the user selects. It's the same thing that you made for Armor mod element...